internal/build/imgsrc/ensure_builder.go

```diff
@@ -73,6 +73,7 @@ func EnsureBuilder(ctx context.Context, org *fly.Organization, region string, re
 					return nil, nil, err
 				}
 			}
+			return builderMachine, builderApp, nil
```

Comment:
If I'm reading this right, this also returns if `err != nil && errors.Is(err, ShouldReplaceBuilderMachine)`, which doesn't seem intended?

Reply:
Yea, I was doing my own review and caught this too! Addressed via https://github.com/superfly/flyctl/pull/3786/commits/f3f196ad2cc4fa8b90246c8829516674fcca1cd8